kolibri/plugins/management/assets/src/state/actions.js

```diff
@@ -362,7 +362,7 @@ function createUser(store, stateUserData) {
     // dispatch newly created user
     newUser => {
       const userState = _userState(newUser);
```

Comment:
@christianmemije I think this might have happened when you came in to make the user management more state-driven